pkg/api/norman/server/managementstored/setup.go

```diff
@@ -532,10 +531,10 @@ func App(schemas *types.Schemas, management *config.ScaledContext, kubeConfigGet
 	schema.Store = store
 	wrapper := app.Wrapper{
 		Clusters:              management.Management.Clusters(""),
+		ClusterManager:        clusterManager,
 		CatalogManager:        management.CatalogManager,
 		TemplateVersionClient: management.Management.CatalogTemplateVersions(""),
 		TemplateVersionLister: management.Management.CatalogTemplateVersions("").Controller().Lister(),
-		KubeConfigGetter:      kubeConfigGetter,
```

Comment:
need to also remove from the wrapper struct right? Since we are effectively replacing KubeConfigGetter with ClusterManager to be able to do more stuff than what the interface allows us to normally do?

Reply:
It's not really being replaced since this wasn't being used at all anyway (although it could be used for what I'm doing), but otherwise yes it's not needed and I have removed from struct.